packages/cinterop/build.gradle.kts

```diff
@@ -179,6 +179,8 @@ kotlin {
             dependencies {
                 implementation(kotlin("stdlib-common"))
                 implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:${Versions.coroutines}")
+
+                api("org.mongodb.kbson:kbson:${Versions.kbson}")
```

Comment:
Does the API visibility propagate all the way to end users if defined in `cinterop`? Or do we also need to define it in `library-base`?

Reply:
Yes,  it propagates down to the end users.